docs/core/tutorials/debugging-with-visual-studio.md

```diff
@@ -140,99 +144,101 @@ To set a conditional breakpoint and test what happens when the user fails to ent
+1. Select **Debug** > **Step Into** or press **F11**. Visual Studio highlights the statement that includes the `currentDate` variable assignment. The **Locals** window shows the value returned by the call to the <xref:System.Console.ReadLine%2A?displayProperty=nameWithType> method. The console window also displays the string entered when the console prompted for input.
 
-1. Select **Debug** > **Step Into** or press the F11 key. The **Autos** window shows the value of the `date` variable after the assignment from the <xref:System.DateTime.Now?displayProperty=nameWithType> property. The console window is unchanged.
+1. Select **Debug** > **Step Into** or press **F11**. The console window is unchanged.
 
-1. Select **Debug** > **Step Into** or press the F11 key. Visual Studio calls the <xref:System.Console.WriteLine(System.String,System.Object,System.Object)?displayProperty=nameWithType> method. The values of the `date` (or `currentDate`) and `name` variables appear in the **Autos** window, and the console window displays the formatted string.
+1. Select **Debug** > **Step Into** or press **F11**. Visual Studio calls the <xref:System.Console.WriteLine(System.String,System.Object,System.Object)?displayProperty=nameWithType> method. The console window displays the formatted string.
 
-1. Select **Debug** > **Step Out** or press Shift and the F11 key. This stops step-by-step execution. The console window displays a message and waits for you to press a key.
+1. Select **Debug** > **Step Out** or press **Shift**+**F11**. This stops step-by-step execution. The console window displays a message and waits for you to press a key.
 
-1. Press any key to close the console window and exit Debug mode.
+1. Press any key to close the console window and stop debugging.
 
 ---
 
-## Building a Release version
+## Build a Release version
 
-Once you've tested the Debug build of your application, you should also compile and test the Release version. The Release version incorporates compiler optimizations that can sometimes negatively affect the behavior of an application. For example, compiler optimizations that are designed to improve performance can create race conditions in asynchronous or multithreaded applications.
+Once you've tested the Debug version of your application, you should also compile and test the Release version. The Release version incorporates compiler optimizations that can sometimes negatively affect the behavior of an application. For example, compiler optimizations that are designed to improve performance can create race conditions in asynchronous or multithreaded applications.
```

Comment:
nit unrelated to your PR (making asynchronous calls that aren't multi-threaded won't introduce race conditions):

```suggestion
Once you've tested the Debug version of your application, you should also compile and test the Release version. The Release version incorporates compiler optimizations that can sometimes negatively affect the behavior of an application. For example, compiler optimizations that are designed to improve performance can create race conditions in multithreaded applications.
```